Obsidian/Client.cs

```diff
@@ -469,34 +500,28 @@ private async Task HandleEncryptionResponseAsync(byte[] data)
         EncryptionEnabled = true;
         minecraftStream = new EncryptedMinecraftStream(networkStream, sharedKey);
 
-        // TODO: Fix compression
-        //await this.SetCompression();
-        await ConnectAsync();
+        this.SendPacket(new LoginSuccess(Player.Uuid, Player.Username)
+        {
+            SkinProperties = this.Player.SkinProperties,
+        });
     }
 
-    // TODO fix compression (.net 6)
+    // TODO fix compression now????
```

Comment:
Do we have a GH issue open for this?

Reply:
That's a good idea we should open an issue for it